django-verdant/rca/static/rca/css/streamfield-blocks/showcase.less

```diff
@@ -0,0 +1,113 @@
+.showcase {
+    &__title {
+        margin: 0;
+
+        @media @media-desktop-small {
+            font-size: @font-size-72;
+        }
+    }
+
+    &__slide {
+        padding-left: @spacing-half;
+        padding-right: @spacing-half;
+    }
+
+    img {
+        width: 100%;
+    }
+
+    &__details {
+        font-weight: 400;
+    }
+
+    &__page-title {
+        font-weight: 700;
+    }
+
+    &__link {
+        display: block;
+        color: @color-highlight-blue;
+    }
+}
+
+// slick styles
+
+.slick-list {
+  overflow: visible; 
+}
+
+.slick-center {
+    //transform: scale(1.25);
```

Comment:
maybe delete the commented stuff? It's always confusing to find

Reply:
it'll be gone in the next version that's coming...